File: Data Structures & Algorithms/number-of-senior-citizens/submission-2.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        res = 0
        for d in details:
            if int(d[11]) >= 6:
                if int(d[11]) == 6:
                    if int(d[12]) > 0:
                        res += 1
                else:
                    res += 1

        return res

File: Data Structures & Algorithms/daily-temperatures/submission-2.java
class Solution {
    public int[] dailyTemperatures(int[] temperatures) {
        int n = temperatures.length;
        int[] res = new int[n];
        for(int i = 0; i < n-1; i++){
            int key = temperatures[i];
            
            for(int j = i + 1; j < n; j++){
                if (temperatures[j] > key){
                    res[i] = (j-i);
                    break;
                }
            }
        }
        return res;
    }
}
